runelite-api/src/main/java/net/runelite/api/Skill.java

```diff
@@ -56,7 +56,8 @@
 	/**
 	 * The level of all skills added together.
 	 */
-	OVERALL("Overall");
+	OVERALL("Overall"),
+	COMBAT("Combat");
```

Comment:
This is a very poor way to implement this feature. This will cause very bad side effects for any plugin which tries to iterate over `Skill.values()` such as the XP tracker plugin. This should instead be incorporated into `Requirement` somehow.

Reply:
wouldn't overall cause issues with that too?, i checked if it interfered with the xp tracker and didnt see an issue.